backend/models.py

```
@@ -32,20 +32,27 @@ class Tab(db.Model):
-            db.session.query(db.func.count(FeedItem.id))
-            .join(Feed)
-            .filter(Feed.tab_id == self.id, FeedItem.is_read == False)
-            .scalar()
-            or 0
-        )
+        if unread_count is None:
+            # Calculate total unread count for all feeds within this tab
+            total_unread = (
+                db.session.query(db.func.count(FeedItem.id))
+                .join(Feed)
+                .filter(Feed.tab_id == self.id, FeedItem.is_read == False)
```

Ruff (0.14.14)

[error] Avoid equality comparisons to `False`; use `not FeedItem.is_read:` for false checks

Replace with `not FeedItem.is_read`

(E712)